vite.main.config.ts

```diff
@@ -21,7 +42,7 @@ export default defineConfig((env) => {
         formats: ['cjs'],
       },
       rollupOptions: {
-        external,
+        external: external.filter((id) => id !== 'openid-client'),
```

Comment:
Electron Forge wants to mark every package in `package.json` as external, but the `openid-client` needs to be bundled because it uses ES modules and electron doesn't support it (in node you can import CJS files from an ES module, but you can't `require` an ES module).